### PR TITLE
Fixed Suihankey info.json

### PR DESCRIPTION
**Pull request by kakunpc**
## Description

Fixed "Info.json".

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/suihankey/info.json [hunk 1]
```diff
@@ -2,11 +2,11 @@
     "keyboard_name": "suihankey",
     "url": "https://kakunpc.booth.pm/",
     "maintainer": "kakunpc",
-    "width": 12,
+    "width": 5,
```
Comment (noroadsleft) [inline keyboards/suihankey/info.json]:
```suggestion
    "width": 6.4,
```

File: keyboards/suihankey/info.json [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
     "height": 4,
```
Comment (noroadsleft) [inline keyboards/suihankey/info.json]:
```suggestion
    "height": 4.85,
```

File: keyboards/suihankey/info.json [hunk 3]
```diff
@@ -2,11 +2,11 @@
...
+            "layout": [{"label":"1", "x":0, "y":0}, {"label":"2", "x":1, "y":0}, {"label":"3", "x":2, "y":0}, {"label":"4", "x":3, "y":0}, {"label":"5", "x":4, "y":0}, {"label":"6", "x":0, "y":1}, {"label":"7", "x":1, "y":1}, {"label":"8", "x":2, "y":1}, {"label":"9", "x":3, "y":1}, {"label":"10", "x":4, "y":1}, {"label":"11", "x":0, "y":2}, {"label":"12", "x":1, "y":2}, {"label":"13", "x":2, "y":2}, {"label":"14", "x":3, "y":2}, {"label":"15", "x":4, "y":2}, {"label":"16", "x":2, "y":3}, {"label":"17", "x":3, "y":3}, {"label":"18", "x":4, "y":3}]
```
Comment (noroadsleft) [inline keyboards/suihankey/info.json]:
```suggestion
            "layout": [{"label":"1", "x":0, "y":0.375}, {"label":"2", "x":1, "y":0.125}, {"label":"3", "x":2, "y":0}, {"label":"4", "x":3, "y":0.125}, {"label":"5", "x":4, "y":0.25}, {"label":"6", "x":0, "y":1.375}, {"label":"7", "x":1, "y":1.125}, {"label":"8", "x":2, "y":1}, {"label":"9", "x":3, "y":1.125}, {"label":"10", "x":4, "y":1.25}, {"label":"11", "x":0, "y":2.375}, {"label":"12", "x":1, "y":2.125}, {"label":"13", "x":2, "y":2}, {"label":"14", "x":3, "y":2.125}, {"label":"15", "x":4, "y":2.25}, {"label":"16", "x":3.4, "y":3.25}, {"label":"17", "x":4.4, "y":3.55}, {"label":"18", "x":5.4, "y":3.85}]
```

**Comment by kakunpc** [comment]
Thanks.

**Comment by noroadsleft** [comment]
@zvecr That might actually be the API being weird. There are two codebases in this folder:

- suihankey (18 keys)
- suihankey/split (36 keys)

This file is for the suihankey; the suihankey split has [its own info.json file](https://github.com/qmk/qmk_firmware/blob/430c37024e8de42bb735f26386f2b13f042e3569/keyboards/suihankey/split/info.json), but it looks like the API may be ignoring that file at the moment.

**Comment by kakunpc** [comment]
Should this be fixed?

**Comment by noroadsleft** [comment]
@kakunpc I think it's fine as-is. I suspect the issue @zvecr has flagged is actually on QMK's end, rather than being something you've done wrong.

**Pull request merged**